csrankings-h.csv

```diff
@@ -55,6 +55,7 @@ Hai-gang Gong,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/2031.htm,NOSCHOLA
 Haibin Cai,Loughborough University,https://www.lboro.ac.uk/departments/compsci/staff/academic-teaching/caihaibin,D7UbzUgAAAAJ
 Haibin Kan,Fudan University,http://www.cs.fudan.edu.cn/?page_id=14441,NOSCHOLARPAGE
 Haibin Ling,Stony Brook University,https://www3.cs.stonybrook.edu/~hling,jWx3qtIAAAAJ
+Haibin Yang,BUPT,https://smp.bupt.edu.cn/info/1154/2118.htm,-AQLKlsAAAAJ

```

Comment:
Mechanical Engineering, not CS.

Reply:
I have remove faculty Haibin Yang from the PR